From 4c84251e17fe65bca6352bcabcd261a1ae93f339 Mon Sep 17 00:00:00 2001
From: Mart Raudsepp <leio@gentoo.org>
Date: Mon, 11 Mar 2019 20:52:53 +0200
Subject: [PATCH 2/3] build: Always install man page

It's already generated and shouldn't be thrown in the same pot as
full gtk-doc generation. It should be either a separate `man` option,
or just always installed.
---
 docs/meson.build | 4 +++-
 meson.build      | 4 +---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/docs/meson.build b/docs/meson.build
index b039cb631..2b1800be6 100644
--- a/docs/meson.build
+++ b/docs/meson.build
@@ -1,3 +1,5 @@
 install_man('nautilus.1')
 
-subdir('reference')
+if get_option('docs')
+  subdir('reference')
+endif
diff --git a/meson.build b/meson.build
index b7c7731dc..f2a94b623 100644
--- a/meson.build
+++ b/meson.build
@@ -237,9 +237,7 @@ subdirs = [
 # Conditional building #
 ########################
 
-if get_option('docs')
-  subdirs += 'docs'
-endif
+subdirs += 'docs'
 if get_option('tests') != 'none'
   subdirs += 'test'
 endif
-- 
2.21.0

